enabler/src/components/payment-methods/applepay.ts

```diff
@@ -95,4 +95,16 @@ export class ApplePayComponent extends DefaultAdyenComponent {
         reject(error);
       });
   }
+
+  isAvailable() {
+    return this.component.isAvailable()
+    .then(() => {
+      console.log(`${this.paymentMethod} is available`);
```

Comment:
Not sure if we should log stuff like this? I noticed multiple log statements in the repo so i added them for now. Let me know

Reply:
Maybe I can deploy to test it works with that, then we can remove before we merge it... 

Regarding the rest of the console log, maybe we need to review and remove.